src/components/bottom-bar/_bottom-bar.scss

```diff
@@ -1,5 +1,5 @@
 .player .bottom-bar {
-  background: linear-gradient(180deg, rgba(0, 0, 0, 0) 0%, rgba(0, 0, 0, 0.6) 100%);
+  background: linear-gradient(180deg, rgba(0, 0, 0, 0.00) 0%, rgba(0, 0, 0, 0.80) 100%);
```

Comment:
why not just rgba(0, 0, 0, 0) ?

Reply:
it's for the gradient, it's started with dark background and in the end it's light.